Command/RemoveThumbsCommand.php

```diff
@@ -236,4 +263,27 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * @return QuestionHelper|DialogHelper
+     */
+    final protected function getQuestionHelper()
```

Comment:
Why did you make this public? IMHO this is just a method for BC reasons. So please make this `private`